src/pages/community-champions.js

```diff
@@ -678,12 +683,33 @@ const communityChampionBreadcrumbs = [
     },
 ];
 
+const ogImage = '/public/images/champions-badge.svg';
+const ogDescription =
+    'Champions are a group of passionate, dedicated advocates of the MongoDB community. They keep the community informed and excited about our latest developments and newest offerings. They are the trusted bridge between MongoDB and our community.';
+
 const CommunityChampions = () => {
-    const metadata = useSiteMetadata();
     const useBannerImageWithSpace = useMedia(screenSize.upToSmall);
+    const metadata = useSiteMetadata();
```

Comment:
```suggestion
    const {siteUrl, title} = useSiteMetadata();
```


If we only need a few things, let's destructure this!